README.md

```diff
@@ -99,7 +99,7 @@ a disk named /dev/sda, you would run the following command to partition, format
 and mount the disk.
 
 ```console
-sudo nix --experimental-features "nix-command flakes" run github:nix-community/disko -- --mode disko /tmp/disk-config.nix
+sudo nix --experimental-features "nix-command flakes" run github:nix-community/disko/v1.8.0 -- --mode disko /tmp/disk-config.nix
```

Comment:
```suggestion
sudo nix --experimental-features "nix-command flakes" run github:nix-community/disko/latest -- --mode disko /tmp/disk-config.nix
```

It would be nice if we had a branch or tag that always pointed to the latest release, you could also use it in your `flake.nix`:

```nix
{
  inputs.disko.url = "github:nix-community/disko/latest";
}